.github/script/backward-compatibility-run-check.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+
+/**
+ * This script simply checks whether the version we are releasing is either the first NPM version for any new major version or
+ * a beta version. For both, we do not want to run backward compatibility checks since both will have backward in-compatible breaking changes.
+ * 
+ * For e.g: 3.0.0, 3.0.0-beta.0, 4.0.0 and so on.
+ */
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const versionToRelease = require('../../package.json').version;
+if (versionToRelease.includes('beta') || (/^[0-9]+\.0\.0$/g).test(versionToRelease)) {
+  console.log('no');
```

Comment:
Can we print out something more meaningful and return a value instead of console.log?

Reply:
+1

Reply:
Do you guys have an example to return a value and catch it in GitHub action run? I see all scripts to be logging to console and accessing through a bash call in GA hence checking. I am re-searching more around this.

Reply:
I think I have to use `process.exit` on the node script and then based on the job success failure run the actual checks.

Reply:
So I discussed with Trung and could not find a way to return a boolean and catch it in GitHub actions, hence, reverted to `console.log` as we have in other scripts. I extended the log though from "yes" or "no" to a detailed log and check it using `contains` in GitHub action `if` conditional.